Domains/4-Application/OpenSite/Released/OpenSite.01.00.04.ecschema.xml

```diff
@@ -0,0 +1,971 @@
+        </Target>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="IslandEdgeOwnsAisleAspect" modifier="Sealed" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="IslandEdge" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="EdgeAisleAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="PadEdgeOwnsAisleAspect" modifier="Sealed" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="PadEdge" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="EdgeAisleAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="LayoutParkingEdgeOwnsAisleAspect" modifier="Sealed" strength="embedding">
+        <BaseClass>bis:ElementOwnsUniqueAspect</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="owns" polymorphic="false">
+            <Class class="LayoutParkingEdge" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="is owned by" polymorphic="false">
+            <Class class="EdgeAisleAspect" />
+        </Target>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="AuthoringElementDrivesPlanElements" modifier="None" strength="referencing">
+        <BaseClass>bis:ElementDrivesElement</BaseClass>
+        <Source multiplicity="(1..1)" roleLabel="creates" polymorphic="true">
+            <Class class="bis:SpatialLocationElement"/>
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="is created by" polymorphic="true">
+            <Class class="spcomp:Space"/>
+        </Target>
+    </ECRelationshipClass>
+
+    <ECStructClass typeName="DistanceAlongLinearPath" description="Describes a distance along a linear path, as either an absolute distance or a percent along" modifier="Sealed">
+        <ECProperty propertyName="AbsoluteAlong" typeName="double" kindOfQuantity="rru:LENGTH_SHORT" displayLabel="Applicable when IsDistanceAbsolute is true. It is the distance along the linear path in meters"/>
+        <ECProperty propertyName="PercentAlong" typeName="double" kindOfQuantity="rru:PERCENTAGE" displayLabel="Applicable when IsDistanceAbsolute is false. It is the percent along the linear path"/>
+        <ECProperty propertyName="IsDistanceAbsolute" typeName="boolean" displayLabel="True if DistanceAlong property should be used, false if PercentAlong describes the distance. Undefined implies an absolute distance"/>
+    </ECStructClass>
+
+    <ECRelationshipClass typeName="ElementDrivesElementsLinearly" modifier="Abstract" strength="referencing" strengthDirection="forward" description="">
```

Comment:
I think the validation is complaining about the `description` attribute so I think you have to either describe something in this attribute or remove it form `ElementDrivesElementsLinearly` and `ElementDrivesDistributionStructuresLinearly` ECRelationship Classes.

Reply:
If that is the case, it would be nice if the tool produced a clearer error message. The current one leaves one with no clue about what's wrong.